scripts/irods/test/test_quotas.py

```diff
@@ -97,3 +97,54 @@ def test_filter_out_groups_when_selecting_user__issue_3507(self):
 
         finally:
             self.admin.assert_icommand(['iadmin', 'rmgroup', 'test_group_3507'])
+
+    def test_physical_quota_case_10_11_12(self):
```

Comment:
Missing issue number suffix.

The cases which the test cover can be captured in a comment at the start of the test rather than the test name.